_includes/404.html

```diff
@@ -14,10 +14,10 @@ <h2 class="section-heading">Error 404 - Page not Found!</h2>
         <p>Were you trying to find one of the following?</p>
         <div style="display: inline-block; text-align: left">
             <ul class="text-muted">
-                <li>What's going on: <a href="#events.html">Events</a></li>
-                <li>Who's talking it up: <a href="#speakers.html">Speakers</a> or <a href="#previous.html">Alumni</a></li>
-                <li>Who's supporting us: <a href="#sponsor.html">Sponsor</a></li>
-                <li>How to help: <a href="#volunteer.html">Volunteer</a></li>
+                <li>What's going on: <a href="http://www.wichitalks.com/#events.html">Events</a></li>
+                <li>Who's talking it up: <a href="http://www.wichitalks.com/#speakers.html">Speakers</a> or <a href="http://www.wichitalks.com/#previous.html">Alumni</a></li>
+                <li>Who's supporting us: <a href="http://www.wichitalks.com/#sponsor.html">Sponsor</a></li>
+                <li>How to help: <a href="http://www.wichitalks.com/#volunteer.html">Volunteer</a></li>
             </ul>
           </div>
         <p>You can also use your browser's back button to return to your previous page or go <a href="http://www.wichitalks.com" target="_blank">home</a> to start again.</p>
```

Comment:
Rats, I thought I got them all. I'll fix the one that sends the user home again. Sorry about that.